### PR TITLE
bump diligent-core dependencies and remove libjpeg for correct libjped-turbo override

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **diligent-core/all**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
which recipe?

**Comment by AndreyMlashkin** [comment]
It's not a direct dependency as I see here
https://github.com/DiligentGraphics/DiligentCore/tree/master/ThirdParty

File: recipes/diligent-core/all/conanfile.py [hunk 1]
```diff
@@ -89,17 +89,12 @@ def build_requirements(self):
     def requirements(self):
         self.requires("opengl/system")
 
-        self.requires("libjpeg/9d")
-        self.requires("libtiff/4.3.0")
-        self.requires("zlib/1.2.11")
-        self.requires("libpng/1.6.37")
-
         self.requires("spirv-cross/cci.20210930")
```
Comment (SpaceIm) [inline recipes/diligent-core/all/conanfile.py:92]: Does it work with `spirv-cross/1.3.204.0`?

**Comment by AndreyMlashkin** [comment]
every time I replace CONAN_PKG:: in patches, I am getting linkage errors... If someone can help here, would be great

**Review by SpaceIm — APPROVED** [review]

**Review by intelligide — APPROVED** [review]

**Comment by AndreyMlashkin** [comment]
@SSE4 @uilianries @jgsogo 
please review

**Review by SSE4 — APPROVED** [review]

**Pull request merged**